yogstation/code/game/objects/items/grenades/glitterbombs.dm

```diff
@@ -0,0 +1,44 @@
+/obj/item/grenade/plastic/glitterbomb
+	name = "plastic party explosive"
+	desc = "Used to put the party in specific locations."
+	can_attach_mob = TRUE
+	var/glitter_type = "glitter"
+
+/datum/effect_system/smoke_spread/chem/glitter
+	effect_type = /obj/effect/particle_effect/smoke/chem/glitter
+
+/obj/effect/particle_effect/smoke/chem/glitter
+	lifetime = 6
+	opaque = TRUE
+
+/obj/item/grenade/plastic/glitterbomb/Initialize()
+	. = ..()
+	create_reagents(20)
+	reagents.add_reagent(glitter_type, 20)
+
+/obj/item/grenade/plastic/glitterbomb/prime()
+	var/datum/effect_system/smoke_spread/chem/glitter/smoke = new()
```

Comment:
lol it's glitter not smoke, stupid. Make a new chem called glitter instead u retrd